File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/EnemyMountainDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class EnemyMountainDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playEnemyMountainDragonKillAnimation() {
        new AnimationTester().testAnimation(new EnemyMountainDragonKillAnimation());
    }
}

File: test/hud/animations/EnemyMountainDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {EnemyMountainDragonKillAnimation} from '../../../src/hud/animations/EnemyMountainDragonKillAnimation.js';

/**
 * Integration test for enemy mountain dragon kill animation on actual hardware.
 */
describe('EnemyMountainDragonKillAnimation Integration', () => {
    test('playEnemyMountainDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new EnemyMountainDragonKillAnimation());
    }, 30000);
});
